docs/features/analysis.md

```diff
@@ -643,3 +643,112 @@ spec:
           value: "Bearer {{ args.api-token }}" 
 ```
 
+## Handling Metric Results - NaN and Infinity
+Metric providers can sometimes return values of NaN (not a number) and infinity. Users can edit the `successCondition` and `failureCondition` fields
```

Comment:
I think we should mention the functions in the docs. Can you explain the fact that we provide two math functions `isNaN` and `isInf` as a convenience to allow users to customize the handling of it and what they return (it may not be obvious to everyone). Also note that if not handled, a `NaN` result could result in Failed measurements.

```
!!! important

    available since v1.0
```